app/(app)/alpha/additional-details/_client.tsx

```diff
@@ -39,169 +39,166 @@ export default function AdditionalSignUpDetails({
   details: UserDetails;
 }) {
   const session = useSession();
-
-  const useFormObject = useForm<TypeAdditionalDetailsSchema>({
-    resolver: zodResolver(AdditionalDetailsSchema),
-    defaultValues: details,
-  });
-
   const searchParams = useSearchParams();
-  const slide = Number(searchParams.get("slide")) || 1;
 
-  const onFormSubmit = async (data: TypeAdditionalDetailsSchema) => {
-    try {
-      const isSuccess = await handleFormSubmit(data);
-      if (isSuccess) {
-        toast.success("Saved");
-      } else {
-        toast.error("Error, saving was unsuccessful.");
-      }
-    } catch (error) {
-      toast.error("An unexpected error occurred.");
-    }
-  };
+  const slide = Number(searchParams.get("slide")) || 1;
   if (!session) {
     return redirect("/get-started");
   }
-
   return (
-    <>
-      <FormProvider {...useFormObject}>
-        <form
-          className="min-h-[41rem]"
-          onSubmit={useFormObject.handleSubmit((data) => onFormSubmit(data))}
-        >
-          <SignupProgressBar currentSlide={slide} />
-
-          {slide === 1 && <SlideOne />}
-          {slide === 2 && <SlideTwo />}
-          {slide === 3 && <SlideThree />}
-          {slide === 4 && <SlideFour />}
-        </form>
-      </FormProvider>
-    </>
+    <div className="min-h-[41rem]">
+      <SignupProgressBar currentSlide={slide} />
+
+      {slide === 1 && <SlideOne details={details} />}
+      {/* {slide === 2 && <SlideTwo details={details} />} */}
+      {/* {slide === 3 && <SlideThree details={details} />} */}
+    </div>
   );
 }
 
-function SlideOne() {
-  const {
-    register,
-    trigger,
-    formState: { errors },
-  } = useFormContext();
-
+function SlideOne({ details }: { details: UserDetails }) {
   const router = useRouter();
 
-  const handleClickNextSlide = async (
-    e: React.MouseEvent<HTMLButtonElement, MouseEvent>,
-  ) => {
-    e.preventDefault();
-    const isError = await trigger(
-      ["firstName", "surname", "username", "location"],
-      {
-        shouldFocus: true,
-      },
-    );
+  const { username, firstName, surname, location } = details;
 
-    if (isError) {
+  const {
+    register,
+    handleSubmit,
+    formState: { errors, isSubmitting },
+  } = useForm<TypeSlideOneSchema>({
+    resolver: zodResolver(slideOneSchema),
+    defaultValues: { username, firstName, surname, location },
+  });
+
+  const onFormSubmit = async (data: TypeSlideOneSchema) => {
+    try {
+      // TODO add server action logic
+      // await handleFormSlideOneSubmit(data);
+      toast.success("Saved");
       router.push(`?slide=${2}`, { scroll: false });
+    } catch (error) {
+      toast.error("An unexpected error occurred.");
     }
   };
 
+  // const onFormSubmit = async (data: TypeSlideOneSchema) => {
```

Comment:
Is this comment needed?

Reply:
> Is this comment needed?

I have two options for the onFormSubmit function. Line 73 vs line 84. 
>>const isSuccess = await handleFormSlideOneSubmit(data)
Returns a boolean from the action.

My preference is for the commented out version, but I wanted your opinion on it.

Reply:
Perfect! I'll look at it a little later. In meetings for the afternoon.

Reply:
Go with your preference! I think that's good for the moment. 🦾